src/index.js

```diff
@@ -46,47 +44,58 @@ module.exports = {
     'id-length': ['error', {
       exceptions: ['_', 'e', 'i']
     }],
+    'id-match': ['error', '^[_a-zA-Z][a-zA-Z0-9]*$|^[A-Z][A-Z]+_[A-Z][A-Z]+$'],
```

Comment:
Ok. I don't have a strong opinion on this also. I just mostly experimenting a bit.